config/api.yml

```diff
@@ -540,6 +540,19 @@
         :identifier: instance_guest_restart
       - :name: reset
         :identifier: instance_reset
+  :notifications:
+    :description: User's past notifications
+    :options:
+    - :collection
+    - :arbitrary_resource_path
```

Comment:
Ok, I found #10941 from Tim. Once that is merged, we can remove `arbitrary_resource_path` from here.